projects/vdk-plugins/vdk-kerberos-auth/setup.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import pathlib
+
+import setuptools
+
+
+__version__ = "0.1.0"
+
+setuptools.setup(
+    name="vdk-kerberos-auth",
+    version=__version__,
+    url="https://github.com/vmware/versatile-data-kit",
+    description="Versatile Data Kit SDK plugin adds Kerberos/GSSAPI support.",
+    long_description=pathlib.Path("README.md").read_text(),
+    long_description_content_type="text/markdown",
+    install_requires=["vdk-core"],
```

Comment:
You need to put all packages that need to be installed when vdk-kerberos-auth is installed. 
So likely we need requests-kerberos and minikerberos

Reply:
Thank you for pointing this out.